go/ct/common/opcodes.go

```diff
@@ -292,8 +292,8 @@ func (op OpCode) String() string {
 		return "TIMESTAMP"
 	case NUMBER:
 		return "NUMBER"
-	case DIFFICULTY:
-		return "DIFFICULTY"
+	case PREVRANDAO:
+		return "PREVRANDAO"
```

Comment:
The code 44 still translates to DIFFICULTY for revisions < Paris,
I guess we should respect the printouts.

Reply:
As discussed the opcode name is not used in any important functionality therefore it is sufficient to just return the new name.